assets/js/manual-run-panel/ManualRunPanel.tsx

```diff
@@ -168,12 +169,23 @@ export const ManualRunPanel: WithActionProps<ManualRunPanelProps> = props => {
     pushEventTo(
       'search-selectable-dataclips',
       { job_id: job_id, search_text: '', limit: 10 },
-      (response: any) => {
-        const dataclips = response.dataclips as Dataclip[];
-        setRecentClips(dataclips);
+      (response: unknown) => {
+        const typedResponse = response as { dataclips: Dataclip[]; next_cron_run_id: string | null };
+        setRecentClips(typedResponse.dataclips);
+        setNextCronRunId(typedResponse.next_cron_run_id);
+        
+        // Auto-select the next cron run dataclip if it exists (for cron jobs)
+        if (typedResponse.next_cron_run_id && !runId) {
+          const nextCronDataclip = typedResponse.dataclips.find(
+            clip => clip.id === typedResponse.next_cron_run_id
+          );
+          if (nextCronDataclip) {
+            selectDataclipForManualRun(nextCronDataclip);
+          }
+        }
```

Comment:
Okay. saw on the elixir end that the cron_run_id dataclip would always be available. cool